consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeArtifacts.java

```diff
@@ -47,16 +48,35 @@ public RoundChangeCertificate getRoundChangeCertificate() {
   public static RoundChangeArtifacts create(final Collection<RoundChange> roundChanges) {
 
     final Collection<SignedData<RoundChangePayload>> payloads =
-        roundChanges
-            .stream()
-            .map(roundChange -> roundChange.getSignedPayload())
-            .collect(Collectors.toList());
+        roundChanges.stream().map(RoundChange::getSignedPayload).collect(Collectors.toList());
+
+    Optional<RoundChange> newestRoundChange = empty();
```

Comment:
you could probably use max or min change your compareNewRound to return an int so can be used as a comparator.

Something like
```
 Optional<RoundChange> newestRoundChange = roundChanges.stream().max(RoundChangeArtifacts::compareNewRound);
...
private static int compareNewRound(
      final RoundChange first, final RoundChange second) {
    final PreparedCertificate firstCert = first.getPreparedCertificate().get();
    final PreparedCertificate secondCert = second.getPreparedCertificate().get();
    return Integer.compare(secondCert.getProposalPayload().getPayload().getRoundIdentifier().getRoundNumber(),
        firstCert.getProposalPayload().getPayload().getRoundIdentifier().getRoundNumber());
  }
```